xml/System.Net/WebClient.xml

```diff
@@ -1612,18 +1612,15 @@
  For more information about how to handle events, see [Handling and Raising Events](/dotnet/standard/events/).  
   
 > [!NOTE]
-> A passive FTP file transfer will always show a progress percentage of zero, since the server did not send the file size. To show progress, you can change the FTP connection to active by overriding the <xref:System.Net.WebClient.GetWebRequest%2A> virtual method:  
+> A passive FTP file transfer will always show a progress percentage of zero, since the server did not send the file size. To show progress, you can change the FTP connection to active by overriding the <xref:System.Web.Services.Protocols.WebClientProtocol.GetWebRequest(System.Uri)> virtual method:  
   
 ```csharp  
-internal class MyWebClient : WebClient
+protected override WebRequest GetWebRequest(Uri address)
```

Comment:
```suggestion
It would be useful to also show the inheritance necessary to make this sample work. Here's what I would use:

    internal class MyWebClient : WebClientProtocol
    {
        protected override WebRequest GetWebRequest(Uri address)
        {
            FtpWebRequest req = (FtpWebRequest)base.GetWebRequest(address);
            req.UsePassive = false;
            return req;
        }
    }
```